lms/djangoapps/instructor/views/api.py

```diff
@@ -3397,6 +3398,29 @@ def update_cohort_assignment(request, course_id):
     return JsonResponse({})
 
 
+@transaction.non_atomic_requests
+@ensure_csrf_cookie
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_global_staff
+@require_http_methods(['POST', 'GET'])
+def cohorts_list_with_assignment(request, course_id):
+    cohort_info = json.loads(cohort_handler(request,course_id).content)
```

Comment:
Missing space, or we can get error in this place `cohort_handler(request,course_id)`. Please, fix it.

Reply:
Done